From randy.dunlap@oracle.com  Fri Jun  5 11:15:30 2009
From: Randy Dunlap <randy.dunlap@oracle.com>
Date: Fri, 05 Jun 2009 11:12:03 -0700
Subject: Staging: vt6655: uses pci functions, should depend on PCI
To: <devel@driverdev.osuosl.org>
Cc: Greg KH <gregkh@suse.de>
Message-ID: <4A295FF3.6060706@oracle.com>


From: Randy Dunlap <randy.dunlap@oracle.com>

This driver uses lots of pci_*() calls, so it should depend on PCI.

drivers/staging/vt6655/device_main.c:3942: error: implicit declaration of function 'pci_dev_driver'

Signed-off-by: Randy Dunlap <randy.dunlap@oracle.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/vt6655/Kconfig |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/staging/vt6655/Kconfig
+++ b/drivers/staging/vt6655/Kconfig
@@ -1,5 +1,6 @@
 config VT6655
    tristate "VIA Technologies VT6655 support"
+   depends on PCI
    ---help---
    This is a vendor-written driver for VIA VT6655.
 
